tests/ci/build_run_benchmarks.sh

```diff
@@ -7,49 +7,46 @@ set -x
+
+cd ..
+
 # run this from the bm_framework root directory!
-OPENSSL_ROOT=$(pwd)/openssl
-BORINGSSL_ROOT=$(pwd)/boringssl
 AWSLC_PR_ROOT=$(pwd)/"${PR_FOLDER_NAME}"
 AWSLC_PROD_ROOT=$(pwd)/aws-lc-prod
 
+NUM_CPU_THREADS=$(nproc)
+
 # clone the various repositories we need (we already have aws-lc-pr since we need it to run this script)
 git clone https://github.com/awslabs/aws-lc.git aws-lc-prod
-git clone https://boringssl.googlesource.com/boringssl
-git clone --branch OpenSSL_1_1_1-stable https://github.com/openssl/openssl.git
-
-# build OpenSSL
-mkdir openssl/build
-(cd openssl && ./config --prefix="${OPENSSL_ROOT}"/build --openssldir="${OPENSSL_ROOT}"/build)
-make -C openssl
-make install -C openssl
-
-# build BoringSSL
-mkdir boringssl/build
-cmake -Bboringssl/build -Hboringssl -GNinja -DCMAKE_BUILD_TYPE=Release
-ninja -C boringssl/build
 
 # build AWSLC pr
-mkdir "${PR_FOLDER_NAME}"/build
+mkdir -p "${PR_FOLDER_NAME}"/build
+mkdir -p "${PR_FOLDER_NAME}"/install
 cmake -B"${PR_FOLDER_NAME}"/build -H"${PR_FOLDER_NAME}" -GNinja -DCMAKE_BUILD_TYPE=Release \
-  -DAWSLC_INSTALL_DIR="${AWSLC_PR_ROOT}" \
-  -DBORINGSSL_INSTALL_DIR="${BORINGSSL_ROOT}" \
-  -DOPENSSL_INSTALL_DIR="${OPENSSL_ROOT}"
+  -DAWSLC_INSTALL_DIR="${AWSLC_PR_ROOT}"/install \
+  -DCMAKE_INSTALL_PREFIX="${AWSLC_PR_ROOT}"/install
 ninja -C "${PR_FOLDER_NAME}"/build
 
 # build FIPS compliant version of AWSLC pr
-mkdir "${PR_FOLDER_NAME}"/fips_build
-cmake -B"${PR_FOLDER_NAME}"/fips_build -H"${PR_FOLDER_NAME}" -GNinja -DFIPS=1 -DCMAKE_BUILD_TYPE=Release -DAWSLC_INSTALL_DIR="${AWSLC_PR_ROOT}"
+mkdir -p "${PR_FOLDER_NAME}"/fips_build
+mkdir -p "${PR_FOLDER_NAME}"/fips_install
+cmake -B"${PR_FOLDER_NAME}"/fips_build -H"${PR_FOLDER_NAME}" -GNinja -DFIPS=1 -DCMAKE_BUILD_TYPE=Release -DAWSLC_INSTALL_DIR="${AWSLC_PR_ROOT}"/fips_install -DBUILD_SHARED_LIBS=TRUE -DCMAKE_INSTALL_PREFIX="${AWSLC_PR_ROOT}"/fips_install
 ninja -C "${PR_FOLDER_NAME}"/fips_build
 
 # build AWSLC prod
-mkdir aws-lc-prod/build
-cmake -Baws-lc-prod/build -Haws-lc-prod -GNinja -DCMAKE_BUILD_TYPE=Release -DAWSLC_INSTALL_DIR="${AWSLC_PROD_ROOT}"
+mkdir -p aws-lc-prod/build
+mkdir -p aws-lc-prod/install
+cmake -Baws-lc-prod/build -Haws-lc-prod -GNinja -DCMAKE_BUILD_TYPE=Release -DAWSLC_INSTALL_DIR="${AWSLC_PROD_ROOT}/install" -DCMAKE_INSTALL_PREFIX="${AWSLC_PROD_ROOT}/install"
 ninja -C aws-lc-prod/build
 
 #build FIPS compliant version of AWSLC prod
-mkdir aws-lc-prod/fips_build
-cmake -Baws-lc-prod/fips_build -Haws-lc-prod -GNinja -DFIPS=1 -DCMAKE_BUILD_TYPE=Release -DAWSLC_INSTALL_DIR="${AWSLC_PROD_ROOT}"
+mkdir -p aws-lc-prod/fips_build
+mkdir -p aws-lc-prod/fips_install
+cmake -Baws-lc-prod/fips_build -Haws-lc-prod -GNinja -DFIPS=1 -DCMAKE_BUILD_TYPE=Release -DAWSLC_INSTALL_DIR="${AWSLC_PROD_ROOT}/fips_install" -DBUILD_SHARED_LIBS=TRUE -DCMAKE_INSTALL_PREFIX="${AWSLC_PROD_ROOT}/fips_install"
 ninja -C aws-lc-prod/fips_build
 
 # avoid cpus 0-3 since there are a lot of other things running on them
```

Comment:
Is this comment still true for codebuild?

Reply:
yeah, I don't think we should be binding processes to cores. Regarding the comment, I'm not sure where that claim comes from, it's weird.